android/test-app-util.gradle

```diff
@@ -232,30 +243,40 @@ ext.getVersionName = {
     return "1.0"
 }
 
+ext.isBridgelessEnabled = { project, isNewArchEnabled ->
+    if (isNewArchEnabled) {
+        def reactBridgelessEnabled = tryGetProperty(project, "react.bridgelessEnabled")
+        def bridgelessEnabled = tryGetProperty(project, "bridgelessEnabled")
+        if (reactBridgelessEnabled == "true" || bridgelessEnabled == "true") {
+            def version = getPackageVersionNumber("react-native", project.rootDir)
+            return version == 0 || version >= v(0, 73, 0)
+        }
+    }
```

Comment:
I wonder if this should be more "similar" to the `isNewArchitectureEnabled` logic a few lines down, where we check if the version support earlier and throw an error/GradleExpection if you are trying to enable bridgeless on something lower than 0.73

Reply:
Made it a warning since you may still want New Arch.
```
% ./gradlew assembleDebug
Configuration on demand is an incubating feature.

> Configure project :
react-native 0.73 or greater is required for Bridgeless Mode — disable `bridgelessEnabled` in your `gradle.properties` or upgrade `react-native`
```